HomeworkOnlineStore/HomeworkOnlineStore/Models/Products.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeworkOnlineStore.Models
+{
+    internal class Product
+    {
+        private double _price;
+        private string _name;
+        private int _productID;
+
+        public Product()
+        {
+            _price = new Random().NextDouble() * 10;
+        }
+
+        public double Price { get { return _price; } }
+        public string Name { get { return _name; } set { _name = value; } }
+        public int ProductID { get { return _productID; } set { _productID = value; } }
```

Comment:
Can be converted into the next format public double Price { get; set; } or public double Price { get; private set; }